docs/sources/setup-grafana/installation/debian/index.md

```diff
@@ -105,6 +105,80 @@ Complete the following steps to install Grafana using DEB or the standalone bina
 1. Depending on which system you are running, click the **Linux** or **ARM** tab on the [download page](/grafana/download).
 1. Copy and paste the code from the [download page](/grafana/download) into your command line and run.
 
+## Install Grafana as a standalone binary
+
+Complete the following steps to install Grafana using the standalone binaries:
+
+1. Navigate to the [Grafana download page](/grafana/download).
+1. Select the Grafana version you want to install.
+   - The most recent Grafana version is selected by default.
+   - The **Version** field displays only tagged releases. If you want to install a nightly build, click **Nightly Builds** and then select a version.
+1. Select an **Edition**.
+   - **Enterprise:** This is the recommended version. It is functionally identical to the open-source version but includes features you can unlock with a license if you so choose.
+   - **Open Source:** This version is functionally identical to the Enterprise version, but you will need to download the Enterprise version if you want Enterprise features.
+1. Depending on which system you are running, click the **Linux** or **ARM** tab on the [download page](/grafana/download).
+1. Copy and paste the code from the [download page](/grafana/download) page into your command line and run.
+1. Create a user account for Grafana on your system:
+
+   ```shell
+   sudo useradd -r -s /bin/false grafana
+   ```
+
+1. Move the unpacked binary to `/usr/local/grafana`:
+
+   ```shell
+   sudo mv <your_grafana_download> /usr/local/grafana
```

Comment:
https://grafana.com/docs/writers-toolkit/write/style-guide/write-for-developers/#placeholder-variables

```suggestion
   sudo mv <DOWNLOAD PATH> /usr/local/grafana
```